### DISCUSSION TITLE
Export all users

### DISCUSSION THREAD

**Discussion by atdcode** [nextcloud/user_migration#384] [Q&A]
Hi 

Thanks for this tool.

Is it possible to export all or multiple users ? 

Thanks.

**Comment by Pytal** [comment]
You may write a script that loops through the output of
```bash
occ user:list
```
and run
```bash
occ user:export <user> <folder>
```
for each user

**Comment by atdcode** [comment]
Thank you @Pytal  for your suggestion.

My problematic is pretty simple but it looks like requiring a lot of effort and i'm lazy.

I just want to migrate users/uid/email/password/settings (not the data) to another instance. 

My purpose is then to integrate it with a brand new ldap server.  

This tools seems to do exactly what i need by exporting .json files with all interesting data which is great . 

I really do not understand why this very nice tool is limited to only the connected user ? it could be a great migration tool tho.

**Reply by come-nc** [reply]
It is not limited to the connected user, the admin can export any user using the command @Pytal suggested. (and import with occ user:import)

**Comment by atdcode** [comment]
Thank you @come-nc  for your answer. I didn't made myself clear but i was talking about the graphical tool not the command `occ:export`  .

**Comment by atdcode** [comment]
Ok i just understood this app add the `occ:export` command. i'm dumb.

**Comment by atdcode** [comment]
Is it possible to export users's passwords as well ? it would be terrific.

**Reply by Pytal** [reply]
Unfortunately this is not currently possible
